physionet-django/user/citi_training_module.py

```diff
@@ -0,0 +1,155 @@
+import requests
+from datetime import datetime
+import xml.etree.ElementTree as ET
+from django.conf import settings
+from django.template import loader
+
+
+def convert_date_time(str_date_time):
```

Comment:
There is something odd about the input argument here. If `str_date_time` is expected to be a string, `str_date_time.text` would raise an error. 

Adding type hints across the PR might be helpful. e.g. this function would look something like:

```
def convert_date_time(str_date_time: str) -> datetime:
```

Perhaps:
1. Change `str_date_time.text` to `str_date_time` here.
2. Change the input arguments (below) to `dteExpiration.text` and `dtePassed.text`

Reply:
Thanks Tom for catching that, I have updated the code for consistency.